proposal/BoWeii/README.rst

```diff
@@ -0,0 +1,115 @@
+Job shop scheduling
+===================
+
+Basic information (including the GitHub repository)
+===================
+
+Job-shop scheduling or the job-shop problem (JSP) is an optimization
+problem in computer science and operations research.
+
+I would try to use genetic-algorithm solve the JSP.
+
+**GitHub repository**: https://github.com/BoWeii/job-shop-scheduling 
+
+Problem to solve
+===================
+In a general job scheduling problem, we are given n jobs **J1, J2, ...,
+Jn** of varying processing times,each job consists of a set of
+operations **O1, O2, ..., On** which need to be processed in a specific
+order (known as precedence constraints). Each operation has a specific
+machine that it needs to be processed on and only one operation in a job
+can be processed at a given time. We need to schedule the jobs on **m**
+machines with varying processing power, while trying to minimize the
+makespan. |image0|
+
+Perspective users
+===================
+
+Some automated processing factories or people with similar needs will
```

Comment:
The size of the problems will determine who can use your tool.  How large do you plan to design for.

Reply:
I think all problem size can be an input, but depend on the number of iteration, it will affect the time spent and whether a good solution can be found.

Reply:
OK, I added it !